frontend/components/modal/MissingTranslations.vue

```diff
@@ -0,0 +1,141 @@
+<script setup lang="ts">
+const { $axios, $missingTranslations } = useNuxtApp()
+const { missingTranslations: { missingTranslationsVisible } } = useModal()
+const { t } = useI18n()
+
+const prefix = 'components/modal/MissingTranslations'
+const existing = useState<Map<string, string>>(`${prefix}.existing`, () => new Map())
+const languages = ['en', 'es', 'fr', 'de']
+const tt = (key: string) => t(`${prefix}.${key}`)
+
+onMounted(async () => {
```

Comment:
FWIW, you could make this work on SSR too, likely with [useFetch](https://nuxt.com/docs/api/composables/use-fetch) (though you could also specify a full path, but that's grosser)

Reply:
Good thought - this will be removed in prod, so there isn't an actual production impact, but I'll keep it in mind!